lib/validation.py

```diff
@@ -35,119 +35,391 @@ def _time_str(string):
             (string, lib.litani.TIME_FORMAT_R)) from e
 
 
-def _get_single_job_arguments():
+# doc-gen
+# {
+#   "page": "litani-run.json",
+#   "order": 1,
+#   "title": "Schema for 'wrapper_arguments' key"
+# }
+def _single_job_schema():
     import voluptuous
+
+    # The *wrapper_arguments* key to run.json maps to the following dict. None
+    # of the values in this dict change at any point during the run; they are
+    # mostly the same as the flags passed to *litani-add-job(1)* for this job.
     return {
         "job_id": str,
+        # A globally-unique ID for this job.
+
         "command": str,
+        # The command that litani will execute in a subshell.
+
         "ci_stage": str,
+        # The name of the 'stage' that this job will execute in, used for
+        # organizing the HTML dashboard.
+
         "verbose": bool,
+
         "timeout_ok": bool,
+        # If true, then if this job times out, the outcome will be set to
+        # 'success'.
+
         "pipeline_name": str,
+        # The name of the 'pipeline' that this job will execute in, used for
+        # organizing the HTML dashboard.
+
         "very_verbose": bool,
+
```

Comment:
Should we add a comment to distinguish `very_verbose` from "regular" `verbose`?

Reply:
These two fields are intentionally undocumented, as they're an implementation detail. (There are a few others, like `status_file` etc). There are various reasons for this, some are only used internally to generate the HTML report, etc.

Reply:
I thought about suppressing them from the man page entirely, but I do use them myself :)